### ISSUE TITLE
telnet: command crashes

### ISSUE THREAD

**Issue by converghub** [theQRL/QRL#347]
When I use "getinfo" and "wallet" commands, I got following errors with the latest codebase.
Here is the log file's link. => https://transfer.sh/O1Nct/qrl.log
```
2017-10-09 13:32:55,932 |synced| INFO : >>> new local connection 1 IPv4Address(TCP, '127.0.0.1', 49186)
Unhandled Error
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/log.py", line 103, in callWithLogger
    return callWithContext({"system": lp}, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/log.py", line 86, in callWithContext
    return context.call({ILogContext: newCtx}, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/context.py", line 122, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/context.py", line 85, in callWithContext
    return func(*args,**kw)
--- <exception caught here> ---
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/posixbase.py", line 597, in _doReadOrWrite
    why = selectable.doRead()
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/tcp.py", line 208, in doRead
    return self._dataReceived(data)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/tcp.py", line 214, in _dataReceived
    rval = self.protocol.dataReceived(data)
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 98, in dataReceived
    if not self.parse_cmd(data):
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 78, in parse_cmd
    self.cmd_mapping[command](args)
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 345, in _wallet
    self.factory.state.state_read_chain(self.factory.chain)
  File "/home/pi/QRL/qrl/core/state.py", line 483, in state_read_chain
    self.zero_all_addresses()
  File "/home/pi/QRL/qrl/core/state.py", line 227, in zero_all_addresses
    self.db.Delete(address)
builtins.AttributeError: 'DB' object has no attribute 'Delete'

2017-10-09 13:32:59,137 |synced| CRITICAL : [TWISTED] Unhandled Error
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/log.py", line 103, in callWithLogger
    return callWithContext({"system": lp}, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/log.py", line 86, in callWithContext
    return context.call({ILogContext: newCtx}, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/context.py", line 122, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/context.py", line 85, in callWithContext
    return func(*args,**kw)
--- <exception caught here> ---
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/posixbase.py", line 597, in _doReadOrWrite
    why = selectable.doRead()
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/tcp.py", line 208, in doRead
    return self._dataReceived(data)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/tcp.py", line 214, in _dataReceived
    rval = self.protocol.dataReceived(data)
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 98, in dataReceived
    if not self.parse_cmd(data):
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 78, in parse_cmd
    self.cmd_mapping[command](args)
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 345, in _wallet
    self.factory.state.state_read_chain(self.factory.chain)
  File "/home/pi/QRL/qrl/core/state.py", line 483, in state_read_chain
    self.zero_all_addresses()
  File "/home/pi/QRL/qrl/core/state.py", line 227, in zero_all_addresses
    self.db.Delete(address)
builtins.AttributeError: 'DB' object has no attribute 'Delete'

2017-10-09 13:33:01,081 |synced| INFO : >>> new local connection 1 IPv4Address(TCP, '127.0.0.1', 49190)
Unhandled Error
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/log.py", line 103, in callWithLogger
    return callWithContext({"system": lp}, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/log.py", line 86, in callWithContext
    return context.call({ILogContext: newCtx}, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/context.py", line 122, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/context.py", line 85, in callWithContext
    return func(*args,**kw)
--- <exception caught here> ---
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/posixbase.py", line 597, in _doReadOrWrite
    why = selectable.doRead()
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/tcp.py", line 208, in doRead
    return self._dataReceived(data)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/tcp.py", line 214, in _dataReceived
    rval = self.protocol.dataReceived(data)
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 98, in dataReceived
    if not self.parse_cmd(data):
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 78, in parse_cmd
    self.cmd_mapping[command](args)
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 551, in _getinfo
    self.output['message'].write('>>> Version: ' + self.factory.chain.version_number + '\r\n')
builtins.AttributeError: 'Chain' object has no attribute 'version_number'

2017-10-09 13:33:04,076 |synced| CRITICAL : [TWISTED] Unhandled Error
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/log.py", line 103, in callWithLogger
    return callWithContext({"system": lp}, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/log.py", line 86, in callWithContext
    return context.call({ILogContext: newCtx}, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/context.py", line 122, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/context.py", line 85, in callWithContext
    return func(*args,**kw)
--- <exception caught here> ---
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/posixbase.py", line 597, in _doReadOrWrite
    why = selectable.doRead()
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/tcp.py", line 208, in doRead
    return self._dataReceived(data)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/tcp.py", line 214, in _dataReceived
    rval = self.protocol.dataReceived(data)
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 98, in dataReceived
    if not self.parse_cmd(data):
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 78, in parse_cmd
    self.cmd_mapping[command](args)
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 551, in _getinfo
    self.output['message'].write('>>> Version: ' + self.factory.chain.version_number + '\r\n')
builtins.AttributeError: 'Chain' object has no attribute 'version_number'

2017-10-09 13:33:05,918 |synced| INFO : >>> new local connection 1 IPv4Address(TCP, '127.0.0.1', 49194)
Unhandled Error
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/log.py", line 103, in callWithLogger
    return callWithContext({"system": lp}, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/log.py", line 86, in callWithContext
    return context.call({ILogContext: newCtx}, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/context.py", line 122, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/context.py", line 85, in callWithContext
    return func(*args,**kw)
--- <exception caught here> ---
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/posixbase.py", line 597, in _doReadOrWrite
    why = selectable.doRead()
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/tcp.py", line 208, in doRead
    return self._dataReceived(data)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/tcp.py", line 214, in _dataReceived
    rval = self.protocol.dataReceived(data)
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 98, in dataReceived
    if not self.parse_cmd(data):
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 78, in parse_cmd
    self.cmd_mapping[command](args)
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 541, in _listaddresses
    addresses, num_sigs, types = self.factory.chain.wallet.inspect_wallet()
builtins.AttributeError: 'Wallet' object has no attribute 'inspect_wallet'

2017-10-09 13:33:20,576 |synced| CRITICAL : [TWISTED] Unhandled Error
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/log.py", line 103, in callWithLogger
    return callWithContext({"system": lp}, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/log.py", line 86, in callWithContext
    return context.call({ILogContext: newCtx}, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/context.py", line 122, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/python/context.py", line 85, in callWithContext
    return func(*args,**kw)
--- <exception caught here> ---
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/posixbase.py", line 597, in _doReadOrWrite
    why = selectable.doRead()
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/tcp.py", line 208, in doRead
    return self._dataReceived(data)
  File "/home/pi/.local/lib/python3.5/site-packages/twisted/internet/tcp.py", line 214, in _dataReceived
    rval = self.protocol.dataReceived(data)
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 98, in dataReceived
    if not self.parse_cmd(data):
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 78, in parse_cmd
    self.cmd_mapping[command](args)
  File "/home/pi/QRL/qrl/core/walletprotocol.py", line 541, in _listaddresses
    addresses, num_sigs, types = self.factory.chain.wallet.inspect_wallet()
builtins.AttributeError: 'Wallet' object has no attribute 'inspect_wallet'
```

**Comment by jleni** [comment]
I have fixes for these two in the grpc branch 
> builtins.AttributeError: 'DB' object has no attribute 'Delete'
> builtins.AttributeError: 'Chain' object has no attribute 'version_number'

Not sure about this one
> builtins.AttributeError: 'Wallet' object has no attribute 'inspect_wallet'
It really looks like a reference to very old code (@cyyber ?)

**Comment by cyyber** [comment]
listaddresses command was disabled and I found that it was re-enabled by @randomshinichi . So I have disabled this again and removed the function as "wallet" command is self sufficient to list addresses. PR is being tested will be committed fixing the above mentioned issues.

**Comment by cyyber** [comment]
Fixed in PR #349
